marconi/app/Main.hs

```diff
@@ -218,16 +245,17 @@ utxoWorker
   :: Coordinator
   -> TChan (ChainSyncEvent (BlockInMode CardanoMode))
   -> FilePath
+  -> Maybe MambaAddresses
   -> IO ()
-utxoWorker Coordinator{_barrier} ch path = Utxo.open path (Utxo.Depth 2160) >>= innerLoop
+utxoWorker Coordinator{_barrier} ch path addresses = Utxo.open path (Utxo.Depth 2160) >>= innerLoop
   where
     innerLoop :: UtxoIndex -> IO ()
     innerLoop index = do
       signalQSemN _barrier 1
       event <- atomically $ readTChan ch
       case event of
         RollForward (BlockInMode (Block (BlockHeader slotNo _ _) txs) _) _ct ->
-          Ix.insert (getUtxoUpdate slotNo txs) index >>= innerLoop
+            Ix.insert (getUtxoUpdate slotNo txs addresses) index >>= innerLoop
         RollBackward cp _ct -> do
```

Comment:
@koslambrou is it possible for Rollback to follow a RollForward?

Reply:
Yep! You normally get a bunch or RollForward events, then a Rollbackward, and so on.

Reply:
in that case it is likely to erroneously report on and Address that will be rolled back in future, right?
I suppose that will be something the Notification service will address? and not an issue for us here?

Reply:
Not an issue for us here